build_cmake/FrameworkOpenGl/CMakeLists.txt

```diff
@@ -6,10 +6,18 @@ set(CMAKE_MODULE_PATH
 	${CMAKE_MODULE_PATH}
 )
 
+include(Header)
+
 set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
-if(UNIX OR WIN32 AND NOT ANDROID AND NOT APPLE AND NOT TARGET_PLATFORM_UNIX_AARCH64)
+if(TARGET_PLATFORM_ANDROID OR TARGET_PLATFORM_IOS OR TARGET_PLATFORM_UNIX_AARCH64)
+	set(USE_GLES ON CACHE BOOL "Use GLES")
+else()
+	set(USE_GLES OFF CACHE BOOL "Use GLES")
+endif()
+
+if(NOT USE_GLES AND NOT APPLE)
```

Comment:
Why `NOT APPLE`? AFAIK, GLEW is used on macOS too.

Reply:
maybe it can be used, but we don't used it ourselves.
its not included when apple(osx) (and linux aarch64)
https://github.com/jpd002/Play-/blob/d065c3443ee4fbb2e40cc706eef6353dcb9cb12c/Source/ui_qt/GSH_OpenGLQt.cpp#L30-L34

Reply:
Oh yeah, you're right.